hector_quadrotor_controller_gazebo/src/quadrotor_hardware_gazebo.cpp

```diff
@@ -59,6 +59,7 @@ bool QuadrotorHardwareSim::initSim(
     std::vector<transmission_interface::TransmissionInfo> transmissions)
 {
   // store parent model pointer
+  gazebo_ros_control::DefaultRobotHWSim::initSim(robot_namespace,model_nh,parent_model,urdf_model,transmissions);
```

Comment:
In PRs, please follow the code format of the projects.
In this case, there should be a space after each comma.

Reply:
Done.